ngraph/frontend/frontend_manager/include/frontend_manager/frontend_exceptions.hpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include "ngraph/check.hpp"
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        class GeneralFailure : public CheckFailure
+        {
+        public:
+            GeneralFailure(const CheckLocInfo& check_loc_info,
+                           const std::string& context,
+                           const std::string& explanation)
+                : CheckFailure(check_loc_info,
+                               "FrontEnd API failed with GeneralFailure: " + context,
+                               explanation)
+            {
+            }
+        };
+
+        class InitializationFailure : public CheckFailure
```

Comment:
I think we need to export these classes.

Reply:
done